config/routes.rb

```diff
@@ -34,11 +34,12 @@
     resources :solutions, only: [:show] do
       resources :iterations, only: [:show]
     end
+    resources :mentors, only: [:index]
   end
 
   # ###### #
   # Mentor #
-  # ###### #
+  # ###### #,
```

Comment:
This is breaking the build.

Reply:
Fixed. Ran the system test file and it appears to be working. I have pushed the changes.